indico/modules/events/papers/client/js/components/RevisionJudgment.jsx

```diff
@@ -0,0 +1,125 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import {useDispatch, useSelector} from 'react-redux';
+import {Button, Confirm, Header} from 'semantic-ui-react';
+
+import {Param, Translate} from 'indico/react/i18n';
+import {serializeDate} from 'indico/utils/date';
+
+import {resetPaperJudgment} from '../actions';
+import {getPaperDetails, getPaperPermissions, isPaperStateResetInProgress} from '../selectors';
+import UserAvatar from './UserAvatar';
+
+export default function RevisionJudgment({revision}) {
+  const {state, judge, isLastRevision, judgmentComment, judgmentDt} = revision;
+  const {event, contribution} = useSelector(getPaperDetails);
+  const [confirmOpen, setConfirmOpen] = useState(false);
+  const {canJudge} = useSelector(getPaperPermissions);
+  const isResetInProgress = useSelector(isPaperStateResetInProgress);
+  const dispatch = useDispatch();
+
+  if (state === 'submitted') {
+    return null;
+  }
+
+  return (
+    <div className="i-timeline-item">
+      <UserAvatar user={judge} />
+      <div className="i-timeline-item-box header-indicator-left">
+        <div className="i-box-header flexrow">
+          <div className="f-self-stretch">
+            {state === 'accepted' && (
+              <Translate>
+                <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> accepted this
+                paper.
+              </Translate>
+            )}
+            {state === 'rejected' && (
+              <Translate>
+                <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> rejected this
+                paper.
+              </Translate>
+            )}
+            {state === 'to_be_corrected' && (
+              <Translate>
+                <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> asked for
+                changes.
+              </Translate>
+            )}{' '}
+            <time dateTime={serializeDate(judgmentDt, moment.HTML5_FMT.DATETIME_LOCAL)}>
+              {serializeDate(judgmentDt, 'LL')}
+            </time>
+          </div>
+          {canJudge && isLastRevision && (
+            <>
+              <div className="hide-if-locked">
+                <a
+                  href="#"
+                  className="i-link icon-remove"
+                  title={Translate.string('Reset judgment')}
+                  onClick={() => setConfirmOpen(true)}
+                />
+              </div>
+              <Confirm
+                header={
+                  <Header icon="warning sign" content={Translate.string('Confirm the operation')} />
```

Comment:
Wouldn't that text make more sense as a child? According to the docs this is perfectly valid:

    <Header icon="...">content here</Header>